packages/osl-operator-bundle-image/resources/config/manifests/bases/logic-operator-rhel8.clusterserviceversion.yaml

```diff
@@ -0,0 +1,299 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
I have some questions about this file, specially as I'm no expert on the operator/bundle code.

**1)** When checking the kiegroup/kogito-serverless-operator there are 2 files with this exact same name:

[1] https://github.com/kiegroup/kogito-serverless-operator/blob/main/config/manifests/prod/bases/logic-operator-rhel8.clusterserviceversion.yaml
[2] https://github.com/kiegroup/kogito-serverless-operator/blob/main/bundle.prod/manifests/logic-operator-rhel8.clusterserviceversion.yaml

Comparing the file you created here with them, it is file [1] that you created here. 
But file [2] has the:
`image: registry.redhat.io/openshift-serverless-1/logic-rhel8-operator@sha256:xxxxx`
which points to the sha256 of the operator and I believe we need that file too, don't we? or the content of them needs somehow to be merged in a single file?


**2)** Why this file is ignored in git (packages/osl-operator-bundle-image/.gitignore), dont we need this file pushed to `generated` folder so it is available when building on OSBS?

Reply:
The "bases" dir serves as the base for the CSV creation. The `operator-sdk` tool uses it to generate the final one. When we create this file, we have to set the `USE_HASH` (or something like that) during the usage. I'll review the script to see if we can pass this via an `OSL_BUNDLE_OPERATOR__useHash` var.

The file is ignored because the tool updates the `createdAt` attribute each time we generate it. Then, CI will complain because the file will be generated during the build again, and this field will differ.

In the upcoming PR, we will push all these files to `osl-images`, including the CSV, since there, we won't check for generation whatsoever.